pumps/mongo.go

```diff
@@ -81,18 +91,20 @@ func (m *MongoPump) Init(config interface{}) error {
 		log.WithFields(logrus.Fields{
 			"prefix": mongoPrefix,
 		}).Info("-- No max batch size set, defaulting to 10MB")
-		m.dbConf.MaxInsertBatchSizeBytes = tenMB
+		m.dbConf.MaxInsertBatchSizeBytes = int(10 * MiB)
```

Comment:
Is the `int()` here necessary? I'm almost certain it isn't, as `10*MiB` should be an untyped integer constant.